### PR TITLE
Banner component added from HMCTS Design System

### PR DESCRIPTION
**Pull request by adamsilver**

### CODE REVIEW

File: src/components/moj-banner/template.njk [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<div class="moj-banner{% if params.type == 'success' %} moj-banner--success{% elif params.type == 'warning' %} moj-banner--warning{% else %} {{- ' ' + params.classes if params.classes}}{% endif %}">
```
Comment (whatterz) [inline src/components/moj-banner/template.njk]: Can you append an attributes object to the main `div` container for consistency?

```
{%- for attribute, value in params.attributes -%} {{ attribute }}="{{ value }}"{% endfor %}
```

File: src/components/moj-banner/template.njk [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  {% elif params.type == 'warning' %}
+  <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25"><path d="M13.6,15.4h-2.3v-4.5h2.3V15.4z M13.6,19.8h-2.3v-2.2h2.3V19.8z M0,23.2h25L12.5,2L0,23.2z"/></svg>
+  {% elif params.type == 'information' %}
+  <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25"><path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8
+	C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z"/></svg>
+  {% endif %}
+
+  <div class="moj-banner__message">
+    {% if params.iconFallbackText %}<span class="moj-banner__assistive">{{params.iconFallbackText}}</span>{% endif %}
+    {{ params.html | safe if params.html else params.text }}
```
Comment (whatterz) [inline src/components/moj-banner/template.njk]: Small change, but include `-` to remove whitespace from the output:

```
{{- params.html | safe if params.html else params.text -}}
```

File: src/components/moj-banner/_banner.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+.moj-banner__message {
+  @include govuk-font($size: 19);
+  color: govuk-colour("black");
+  display: block;
+  overflow: hidden;
+}
+
+.moj-banner__message {
+  h2 {
```
Comment (whatterz) [inline src/components/moj-banner/_banner.scss]: You have declared `.moj-banner__message` twice. Consider combining.

File: src/components/moj-banner/_banner.scss [hunk 4]
```diff
@@ -0,0 +1,59 @@
+/* ==========================================================================
+   #BANNER
+   ========================================================================== */
+
+.moj-banner {
```
Comment (whatterz) [inline src/components/moj-banner/_banner.scss]: Consider refactoring:

```
.moj-banner {
  ...

  &__icon {
    ...
  }

  &__message {
    ...
  }

  &__assistive {
    ...
  }
}
```
And so on...

Comment (adamsilver) [reply src/components/moj-banner/_banner.scss]: We decided at HMCTS that it was actually easier to read as it is and to lean on SASS minimally. One to discuss for later and maybe with GOV.UK Frontend peeps.

**Pull request merged**